CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Changes by Version
 
 ##### Breaking Changes
 
+- The traces related metrics on collector now have a new tag `sampler_type` ([#1576](https://github.com/jaegertracing/jaeger/pull/1576))
```

Comment:
Can we list all the affected metrics? Eg grep prom endpoint for sampler_type

Reply:
@guanw ping? We would like to release this week.

Reply:
sry was working on something else and got carried. away. Will update shortly